scrapi/processing/storage.py

```diff
@@ -12,6 +12,7 @@
 from scrapi.processing.base import BaseProcessor
 
 
+
```

Comment:
PEP-E303 too many blank lines (3). 
Python style guides recommend 2 newlines between imports and the rest of the logic. We do perform automated code quality checks with the tool ```flake8```. Unfortunately, this does mean that your build will fail in cases where the style guide isn't followed. I'll comment throughout the rest of the repository where style errors are found, but to get the list of errors you can also run ``` flake8 .``` from the scrapi project to get the errors locally. If you would rather not fix these errors, let me know and I can pull in your branch and make the changes myself.